File: app/src/main/java/com/ramez/shopp/Activities/NoBalanceActivity.java
package com.ramez.shopp.Activities;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.ramez.shopp.R;
import com.ramez.shopp.databinding.ActivityNoBalanceBinding;

public class NoBalanceActivity extends ActivityBase {
    ActivityNoBalanceBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityNoBalanceBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);

        binding.toolBar.mainTitleTxt.setText(R.string.add_balance);
        binding.toolBar.backBtn.setOnClickListener(view1 -> {
            onBackPressed();
        });
        binding.addBalanceBut.setOnClickListener(view1 -> {

        });

    }

    private void startNoBalanceActivity(){
        Intent intent=new Intent(getActiviy(), NoBalanceActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/ramez/shopp/activities/NoBalanceActivity.kt
package com.ramez.shopp.activities


import android.content.Intent
import android.os.Bundle
import android.view.View
import com.ramez.shopp.R
import com.ramez.shopp.databinding.ActivityNoBalanceBinding


class NoBalanceActivity : ActivityBase() {
    var binding: ActivityNoBalanceBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityNoBalanceBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
        binding!!.addBalanceBut.setOnClickListener { view1 -> }
        setTitle(R.string.add_balance)
    }

    private fun startNoBalanceActivity() {
        val intent = Intent(activity, NoBalanceActivity::class.java)
        startActivity(intent)
    }
}